lxd/container_delete.go

```diff
@@ -71,8 +30,14 @@ func containerDelete(d *Daemon, r *http.Request) Response {
 		return SmartError(err)
 	}
 
+	// TODO: i have added this not sure its a good idea (pcdummy)
```

Comment:
So looking at the rest of the code, I think we should still be able to cleanup a container path when the DB entry is missing (so on disk path exists but newLxdContainer fails).